utils/efibootmgr/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=efibootmgr
+PKG_VERSION:=18
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/rhboot/efibootmgr.git
+PKG_SOURCE_DATE:=2022-11-12
+PKG_SOURCE_VERSION:=1904f9cd5a48207d49e393ac8de824f54ccfb697
+PKG_MIRROR_HASH:=0355e00bb54d468ecbaa106aa703dd389f2f2f4c7b7afb78a258cec10d75f78d
+
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=COPYING
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+PKG_MAINTAINER:=OSkari Rauta <oskari.rauta@gmail.com>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/efibootmgr
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Linux user-space application to modify the Intel Extensible Firmware Interface (EFI) Boot Manager
```

Comment:
I think it would be too long for menuconfig, can you try to shorten it?

Reply:
Sure thing